ci-operator/config/maistra/api/maistra-api-maistra-2.3.yaml

```diff
@@ -34,15 +34,14 @@ tests:
   commands: |
     ./maistra/run-test-infra-script.sh ./tools/automator.sh \
     -o maistra \
-    -f /creds-github/github-token \
+    -f /creds-github/token \
     -r istio \
     -c UPDATE_BRANCH=$AUTOMATOR_SHA bin/update_maistra_deps.sh \
```

Comment:
remove the UPDATE_BRANCH variable:

`-c bin/update_maistra_deps.sh`

also for the 2.4 job below.